Classes/CalcsTab.lua

```diff
@@ -390,36 +390,44 @@ end
 
 -- Build the calculation output tables
 function CalcsTabClass:BuildOutput()
-	self.powerBuildFlag = true
+	-- There's a kind of feedback loop between the calc tab and config tab
+	-- The values in the config tab affect the calcs, which then in turn affect the config
+	-- Doing this twice allows the values to "stabalize"
+	for i = 1,2 do
```

Comment:
This strikes me as an in-optimal solution that is likely to cause problems down the road.

Reply:
Do you have a suggestion for how else to resolve this issue?

Reply:
I have removed the more general changes to calc/config which weren't strictly necessary to get Rapid Infusion to work, which includes the code above that you had issues with.